.github/workflows/appimage/Dockerfile

```
@@ -1,24 +1,25 @@
-FROM ubuntu:jammy
+FROM ubuntu:noble
+
+# Force older pipx to use global location.
+ENV PIPX_HOME=/opt/pipx PIPX_BIN_DIR=/usr/local/bin PIPX_MAN_DIR=/usr/local/share/man
 
 RUN apt-get update -q -y \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)